bundles/core/org.eclipse.smarthome.core/src/main/java/org/eclipse/smarthome/core/common/registry/Registry.java

```diff
@@ -49,15 +54,15 @@
      *            key of the element
      * @return element or null if no element was found
      */
-    public E get(K key);
+    public @Nullable E get(K key);
```

Comment:
You can change the return value from `@Nullable` of a parent class to `@Nonnull` in a child but you can't change from `@Nonnull` to `@Nullable`.

It is the same logic used for more specialized returned classes used by the JVM.